grr/server/grr_response_server/gui/ui/lib/api/http_api_service.ts

```diff
@@ -215,6 +215,17 @@ export class HttpApiService {
     return this.http.post<ApiExplainGlobExpressionResult>(url, args).pipe(
         map(result => result.components ?? []));
   }
+
+  addClientLabel(clientId: string, label: string): Observable<ApiAddClientsLabelsArgs> {
+    const url = `${URL_PREFIX}/clients/labels/add`;
+    return this.http.post<ApiAddClientsLabelsArgs>(url, {client_ids: [clientId], labels: [label]});
```

Comment:
The generic type parameter in angle brackets specifies the return type, not the argument type. This API handler does not return a real payload (it only returns `{status: 'ok'}`), so you can use `{}` to specify that the HTTP response contains an object with unknown properties.

Reply:
That was a novice mistake, I must say